backend/src/main/scala/cromwell/backend/model/MemorySize.scala

```diff
@@ -1,4 +1,4 @@
-package cromwell.backend.validation
+package cromwell.backend.model
```

Comment:
Model package is commonly use in most of programming languages to put all VOs related to that component. The idea comes from MVC. 
i.e.: https://github.com/broadinstitute/rawls/tree/develop/src/main/scala/org/broadinstitute/dsde/rawls/model
https://github.com/broadinstitute/firecloud-orchestration/tree/develop/src/main/scala/org/broadinstitute/dsde/firecloud/model